### ISSUE TITLE
ES6 destructuring not properly working

### ISSUE THREAD

**Issue by danielo515** [mohseenrm/lit-it#3]
Hello,

Thanks for this great extension. However, I'm experiencing problems while documenting functions that use destructuring on it's signature. Something like this:

```js
/**
* @function connect
* @param  {type} url                 {description}
* @param  {type} { USE_SSL           {description}
* @param  {type} SSL                 {description}
* @param  {type} logger = console  } {description}
* @return {type} {description}
*/
const connect = ( url , { USE_SSL , SSL , logger = console  }) => {
```
Seems that it does not understand default values neither
Regards

**Comment by mohseenrm** [comment]
With the new ES features, it must have picked up the wrong argument signature. Do you want to send a PR?

**Comment by Pandeyjidev** [comment]
@mohseenrm working on this

**Comment by mohseenrm** [comment]
Fixed with PR #14
